sources/quest_1/chapter_1/step_8/step_8_target.move

```diff
@@ -0,0 +1,13 @@
+module robinson::my_shore {
+
```

Comment:
You should import lib on the top of the module 

```
module robinson::my_shore {

    use  std::signer;

    struct GlobalData has key {
        nb_tree: u8,
        has_river: bool,
        shore_location: address,
        nb_trees: vector<u8>,
    }

    fun shores(s: signer) {
        assert!(signer::address_of(&s) == @0x42, 0);
    }
}
```

Reply:
Noted!